Classes/Domain/Exporter/CsvExporter.php

```diff
@@ -39,11 +40,28 @@ public function setOptions(array $options): void
      * @throws \League\Csv\CannotInsertRecord
      */
     public function compileAndSend(iterable $formDataItems): void
+    {
+        $this->compileCsv($formDataItems)->output($this->fileName);
+    }
+
+    public function compileAndSave(iterable $formDataItems, string $filePath): void
+    {
```

Comment:
Small Nitpick: I think a comment would be useful here, so that one knows that an exception can be thrown here.